connector/connect/client/jvm/pom.xml

```diff
@@ -62,6 +62,30 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.apache.spark</groupId>
+      <artifactId>spark-mllib-common_${scala.binary.version}</artifactId>
+      <version>${project.version}</version>
+      <scope>provided</scope>
```

Comment:
I am pretty new to pom here so just try to understand:

this means that the scala client needs runtime dependencies (jars) in class path on `spark-mllib-common` and `spark-mllib-local`?

Reply:
yes, scala client will depend on both `spark-mllib-common` and `spark-mllib-local`; and the server will depend on `spark-mllib`